app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,49 @@
+footer {
+  display: flex;
+  align-items: center;
+  background-color: rgb(232, 230, 230);
+  width: 95%;
+  height: 200px;
+  border-radius: 15px;
+  margin: 0 auto;
+  margin-bottom: 15px;
+  padding: 15px;
+
+  .icon{
+    margin: 0 auto;
+    color: black;
+
+    &:hover {
+      color: #55679b;
+    }
+  }
+
+  .description {
+    margin: 0 auto;
```

Comment:
Add `text-align: center;` and then you can remove it from the a and the h3.
You can also remove the margin 0 auto from the h3